lab_1_classify_by_unigrams/main.py

```diff
@@ -49,6 +75,13 @@ def create_language_profile(language: str, text: str) -> dict[str, str | dict[st
 
     In case of corrupt input arguments, None is returned
     """
+    if (type(language) is not str) or (type(text) is not str):
+        return None
+
+    language_prof = {"name": language,
+                     "freq": calculate_frequencies(tokenize(text))
```

Comment:
what if calculate_ferquencies will return None?

Reply:
I added this check, done